src/components/List.tsx

```diff
@@ -55,7 +55,7 @@ type TItemDraggable = {
 export type ItemData = SwitchItem | ButtonItem | TextButtonItem | DraggableItem;
 
 export type SwitchItem = {
-	type: EItemType.switch;
+	type: EItemType;
```

Comment:
Avoid changing this type if possible, see other suggestion below to fix the type error.

```suggestion
	type: EItemType.switch;
```